### DISCUSSION TITLE
Overriding default config ( request timeout ) for a request.

### DISCUSSION THREAD

**Discussion by rachitpant** [ElMassimo/js_from_routes#34] [Q&A] [answered]
I have an automatically generated path , for which i want to change the default axios timeout period. Is that possible. There are several automatically generated routes for this resource ( all in a single file ) , but i just want to overwrite the timeout for one of these routes.

**Comment by ElMassimo** [comment, marked as answer]
You should be able to pass the [`timeout` option](https://github.com/axios/axios#config-order-of-precedence) to Axios when [calling the helper](https://github.com/ElMassimo/js_from_routes/blob/main/packages/axios/src/index.ts#L27) for that route.

If you need to call that helper in several places, you could encapsulate the call using a function, to ensure it's always called using the same timeout.

**Reply by rachitpant** [reply]
Thank You.
